react-app/src/api/cosmosAPI.ts

```diff
@@ -14,6 +14,7 @@ const GAS_ADJUSTMENT = 1.3;
 
 interface ICosmosAPI {
   getBalance(): Promise<BigNumberCoin>;
+  getBalanceStaked(): Promise<BigNumberCoin>;
```

Comment:
I know they are using this name but I think `getStakedBalance` sounds better :sosad: 
does the original api return only one kind of coin?

Reply:
> does the original api return only one kind of coin?

Yes, it will throw an error if summing coins of different denom

https://github.com/cosmos/cosmjs/blob/d59dcbc095e797b4bda21057603262dc8c63875c/packages/amino/src/coins.ts#L71-L80